src/trio/_core/_parking_lot.py

```diff
@@ -234,6 +269,23 @@ def repark_all(self, new_lot: ParkingLot) -> None:
         """
         return self.repark(new_lot, count=len(self))
 
+    def break_lot(self, task: Task) -> None:
```

Comment:
I think taking `task` optionally might be a nicer interface (cause we're making this public I guess), defaulting it to `trio.lowlevel.current_task()`.